coveralls.json

```diff
@@ -1,7 +1,8 @@
 {
   "coverage_options": {
     "minimum_coverage": 100,
-    "treat_no_relevant_lines_as_covered": true
+    "treat_no_relevant_lines_as_covered": true,
+    "html_filter_full_covered": true
```

Comment:
Interesting!

Reply:
Just removes all the files that are fully covered. No need to look at them. :)